main.js

```diff
@@ -0,0 +1,109 @@
+
+// var pegs = 3;
+// var discs = 5;
+
+const board = [];
+
+function startGame(board) {
+this.board = [
+  [5, 4, 3, 2, 1],
+  [],
+  []
+  ];
+console.log(' ');
+console.log('TOWERS OF HANOI');
+console.log(' ');
+console.log('--- ' + this.board[0].join(' '));
+console.log('--- ');
+console.log('--- ');
+console.log(' ');
+};
+
+
+
+
+function moveDisc(a, b) { 
+  a -= 1;
+  b -= 1;
+  
+  var isLegal = checkLegal(a, b); //should return boolean
+  if (isLegal === false) { 
```

Comment:
Like Brian mentioned, you should just use the variable because a condition needs to evaluate to be truthy or falsy - a boolean value meets this condition already so no need to do a comparison.